CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# How to contribute to Delta Kernel Rust
+
+Welcome!  We'd love to have you contribute to Delta Kernel Rust!
+
+## Did you find a bug?
+
+Create an issue with a reproducible example.  Please specify the Rust version, delta-kernel-rs version, the code executed, and the error message.
+
+## Did you create a PR to fix a bug?
+
+Open a pull request and add "Fixes #issue_number" in the PR description.
+
+We appreciate bug fixes - thank you in advance!
+
+## Would you like to add a new feature or change existing code?
+
+If you would like to add a feature or change existing behavior, please make sure to create an issue and get the planned work approved by the core team first!
+
+Always better to get aligned with the core devs before writing any code.
+
+## Do you have questions about the source code?
+
+Feel free to create an issue or join the Delta Lake Slack channel with questions!
```

Comment:
link slack channel?

Reply:
We unfortunately can't provide a link cause Slack channel links die every 30 days 🙃

Reply:
or rather the slack workspace in general? can we have the overall slack workspace link?

Reply:
delta-users.slack.com right?

```suggestion
Feel free to create an issue or join the Delta Lake Slack channel (`#delta-kernel` on delta-users.slack.com) with questions!
```